### PR TITLE
Assert SendUserMessage ports

### PR DESCRIPTION
**Pull request by Domiii**
* https://linear.app/replay/issue/RUN-2188/mismatched-portlock-#comment-0e3b1ce4

### CODE REVIEW

File: mojo/core/ports/node.cc [hunk 1]
```diff
@@ -876,9 +880,8 @@ int Node::OnObserveProxy(const PortRef& port_ref,
     }
   }
 
-  if (event_to_forward) {
+  if (event_to_forward)
     delegate_->ForwardEvent(event_target_node, std::move(event_to_forward));
-  }
```
Comment (toshok) [inline mojo/core/ports/node.cc:881]: why this change?  format-on-save?

**Review by toshok — APPROVED** [review]
lgtm, but can you add the braces back in to reduce the LoC churn here before landing?

File: mojo/core/ports/node.cc (hunk 1 above)
Comment (Domiii) [reply mojo/core/ports/node.cc:881]: This was not in the vanilla version. We added it at some point, so I removed it to reduce conflicts 🤷‍♀️
